src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,176 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private final RegistrationService registrationService = new RegistrationServiceImpl();
+    private User validUser = new User();
+
+    @BeforeEach
+    void setUp() {
+        validUser.setLogin("User777");
+        validUser.setAge(25);
+        validUser.setPassword("7771651684");
+    }
+
+    @AfterEach
+    void tearDown() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void register_successfully_Ok() {
+        User actual = registrationService.register(validUser);
+        assertNotNull(actual);
+        StorageDao storageDao = new StorageDaoImpl();
+        String actualLogin = storageDao.get(validUser.getLogin()).getLogin();
+        assertEquals(validUser.getLogin(), actualLogin);
+    }
+
+    @Test
+    void register_userAlreadyExists_NotOk() {
+        User checkingSameLogin = new User();
+        checkingSameLogin.setLogin("User777");
+        checkingSameLogin.setAge(37);
+        checkingSameLogin.setPassword("7659743");
```

Comment:
you can just reuse your valid user, or at least use `setLogin(validUser.getLogin())` to avoid the need to change it in two places if needed